plugins/jobs/web_client/views/JobListWidget.js

```diff
@@ -260,7 +307,33 @@ var JobListWidget = View.extend({
             filter.statuses = JSON.stringify(this.statusFilter);
         }
         this.collection.params = filter;
+
         return this.collection.fetch({}, true);
+    },
+
+    _cancelJobs: function () {
+        Promise.all(
```

Comment:
Per the consensus in #2068, we need to stick with jQuery Promise until at least Girder 3.